src/main/java/com/adq/jenkins/xmljobtodsl/dsl/strategies/custom/DSLParamStrategy.java

```diff
@@ -20,10 +20,19 @@ public String toDSL() {
 	}
 
 	public String getOrderedChildrenDSL() {
-		String defaultValue = getChildrenByName("defaultValue").toDSL();
-		String name = getChildrenByName("name").toDSL();
-		String description = getChildrenByName("description").toDSL();
-		return name + ", " + defaultValue + ", " + description;
+		PropertyDescriptor propertyDescriptor = (PropertyDescriptor) getDescriptor();
+		if (propertyDescriptor.getName().equals("hudson.model.StringParameterDefinition")) {
+			String defaultValue = getChildrenByName("defaultValue").toDSL();
+			String name = getChildrenByName("name").toDSL();
+			String description = getChildrenByName("description").toDSL();
+			return name + ", " + defaultValue + ", " + description;
+
+		} else if (propertyDescriptor.getName().equals("org.jenkinsci.plugins.credentialsbinding.impl.StringBinding")) {
+			String variable = getChildrenByName("variable").toDSL();
+			String credentialsId = getChildrenByName("credentialsId").toDSL();
+			return variable + ", " + credentialsId;
+		}
+		return "";
```

Comment:
Prob want to leave the else condition as it was before so there aren't unintended consequences

Reply:
Do you mean remove the else-if and just have an else statement? This method was specific to one property before so it only had one return statement and no else condition.

Reply:
yeah just change the else if statement on line 30 to be a plain else. I don't think we want to default to a blank string if it's not one of those two conditions